orc8r/gateway/python/magma/ctraced/BUILD.bazel

```diff
@@ -0,0 +1,49 @@
+# Copyright 2022 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@python_deps//:requirements.bzl", "requirement")
+load("@rules_python//python:defs.bzl", "py_binary")
+
+ORC8R_ROOT = "../../"
+
+LTE_ROOT = ORC8R_ROOT + "../../../lte/gateway/python"
+
+DEPS = [
+    "//orc8r/gateway/python/magma/common:sentry",
+    "//orc8r/protos:mconfigs_python_proto",
+    "//orc8r/protos:mconfig_python_proto",
+    "//orc8r/gateway/python/magma/common:service",
+    "//orc8r/protos:ctraced_python_proto",
+    "//orc8r/protos:ctraced_python_grpc"
+]
+
+SRCS = [
+    "command_builder.py",
+    "main.py",
+    "rpc_servicer.py",
+    "trace_manager.py",
+]
+
+
+py_binary(
+    name = "ctraced",
+    srcs = SRCS,
+    imports = [
+        LTE_ROOT,
+        ORC8R_ROOT,
+    ],
+    # legacy_creat_init = False is required to fix issues in module import, see https://github.com/rules-proto-grpc/rules_proto_grpc/issues/145
+    legacy_create_init = False,
+    main = "main.py",
+    python_version = "PY3",
+    visibility = ["//visibility:public"],
+    deps = DEPS,
```

Comment:
let's just embed DEPS and SRCS... (We're planning on cleaning up the rest of services soon)